### DISCUSSION TITLE
下载模组时自动检测模组适配性并给予提示

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#972] [提问] [answered]
在拖动安装模组、启动器内下载模组时自动检测模组适配性并给予提示
（例:尝试在Forge平台安装仅Fabric模组时弹窗提示）
这样能在安装未注明平台的模组时免去启动一次、报错、关闭游戏、安装其他平台的繁琐步骤，也能在安装大量模组时获知哪些模组不适配当前MC版本

（注：我没有在其他discussion中看见相似的提问，如果重复了请接受我的道歉。）

**Comment by WForst-Breeze** [comment, marked as answer]
您的意思或许是：执行模组安装操作时检测该模组与模组加载器是否兼容
1. 拖拽安装检测：如有新功能建议，请提交 [**新功能提案**](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)
2. 启动器内下载检测：现在的 PCL2 会在用户下载 mod 时自动选择与 mod 加载器对应的版本（即 #94 ）或许这已经足够了qwq
